po/fr.po

```diff
@@ -81,7 +81,7 @@ msgstr "Vue des applications en mode sombre"
 #. John Doe https://example.com
 #: src/Application.vala:157
 msgid "translator-credits"
-msgstr ""
+msgstr "Teamcons - teamcons.carrd.co"
```

Comment:
`teamcons.carrd.co` seems to be your portfolio website but this results a normal string at the moment:

![image](https://github.com/user-attachments/assets/d29a950a-6110-46bd-911c-90e67622b614)

You can make this a clickable button that you can access to your website directly with the following change if you want. What do you want to do?
(This is not required, I'm fine either if you don't accept my proposal)

```suggestion
msgstr "Teamcons https://teamcons.carrd.co"
```

![image](https://github.com/user-attachments/assets/940ad0b3-964a-4d38-a135-435683474a27)